modules/profile/manifests/rpmrepo.pp

```diff
@@ -1,3 +1,26 @@
 class profile::rpmrepo {
-  include rpmrepo
+  class { 'rpmrepo':
+    repo_domain => 'repo.if083',
+    repo_name   => 'Our local repopository',
+    repo_dirs   => ['soft','apps'],
+  }
+
+  $repo_conf = @(CONF)
```

Comment:
How about creation of defined type instead of passing whole template as a string....?

Reply:
This config is so short and used only once, so I decided to insert in such simply way. But I understand  that special defined type will be very useful.